### PR TITLE
SDC Library Updated as Per Latest Code to Android Fhir Sdk

### PR DESCRIPTION
**Pull request by MuhammadSalman-7214**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes https://github.com/opensrp/fhircore/issues/398

**Type**
Choose one: Release

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/app/src/main/java/org/smartregister/fhircore/activity/RecordVaccineActivity.kt [hunk 1]
```diff
@@ -169,3 +170,13 @@ class RecordVaccineActivity : MultiLanguageBaseActivity() {
     alertDialog.show()
   }
 }
+
+object Mapper {
```
Comment (pld) [inline android/app/src/main/java/org/smartregister/fhircore/activity/RecordVaccineActivity.kt:174]: why is this necessary?

Comment (MuhammadSalman-7214) [reply android/app/src/main/java/org/smartregister/fhircore/activity/RecordVaccineActivity.kt:174]: It is not necessary. I am removing it.

**Review by pld — APPROVED** [review]
